index.rst

```diff
@@ -46,77 +46,233 @@
+Most of the coordination and system details regarding run coordination are contained in an associated Confluence page night log, this is discussed in a following section.
+Of course, all tasks (or Jira stories) that are planned for execution during this block are to be linked to the epic.
+
+
+
+Proposing an Observation Activity
+=================================
+
+There are numerous types of observing activities, however, they are often categorized into scientific campaigns (e.g. imaging or spectral surveys) and engineering tasks.
+
+- *Campaigns* are meant to indicate a broad set of observations to study system performance or take new science data.
+  They often consist of a survey of spectral and/or imaging data, and the visits are coordinated by the Scheduler.
+  Although not required, campaigns are generally multi-night studies and not one-off characterizations.
+- *Engineering Tasks* encompass activities that are meant to be close to single-occurrence observations.
+  This includes functionality testing and characterization studies (e.g. AOS LUT generation), but can also include discretized science observations that are limited in scope and duration.
+  These tasks are generally run in a more manual fashion, either via the execution of SAL scripts via the scriptQueue CSC, or when required, via a Jupyter notebook running off of Nublado.
+
+The proposal of an observing activity is done via a Jira ticketing process.
+
+Jira Ticket Creation
+--------------------
+
+The jira ticket for a given observation contains all the information to execute an observation, as well as a justification for the observation.
+It is acceptable to attach small files and/or link other documents where applicable, specifically in regards to the justification.
+The prioritization of the observation is handled by the commissioning leads, but all tasks must go through proper preparations to maximize on-sky efficiency. 
+This is discussed further in the `Observation Preparation`_ section.
+
+When proposing an observation, a jira ticket (story), in the `SITCOM Jira Project <https://jira.lsstcorp.org/projects/SITCOM/issues/SITCOM-310?filter=allopenissues>`_.
+Be sure to complete the following steps:
+
+#. Link the epic in which the proposal is to performed
+#. Add a brief description of the observation, and a justification.
+   Linking supporting documentation is encouraged.
+#. Include condition constraints, specifically seeing, transparency, and an evaluation of stability (if appropriate).
+#. Include details on any required supporting hardware (e.g. the DIMM must be operational)
+#. Set the reviewer to ``pingraham``, *and also them add as a watcher*. 
+   The ticket creator can also add themselves as a reviewer if desired.
+#. For Scheduler driven observations, a configuration file will need to be generated.
+   This will be done under a separate ticket and linked here.
+   Contact Tiago Ribeiro for assistance and details.
```

Comment:
I think my previous question about "reviewer" might be applicable here as well. But I think you may want to leave both as is, since the document is reflecting what we do now.. 

That being said, it could be good to add some ideas about how we would do this moving forward.

Reply:
I added a note.

"  The previous procedure is known to suffer from multiple weaknesses. 
  Fixing this structure will be one of the primary tasks and revisions to this document.  "